4-nodarbiba/lekcija/objects/js/app.js

```diff
@@ -1,63 +1,68 @@
-// Objekts ļauj vienā mainīgajā saglabāt datu struktūru ar dažādiem datu tipiem. 
+// Objekts ļauj vienā mainīgajā saglabāt datu struktūru ar dažādiem datu tipiem.
 // Katrai objekta īpašībai ir nosaukums un vērtība - key/value pair
 const person = {
-    firstName: "Elon", // string
-    lastName: "Musk",
-    age: 50, // number
-    companiesFounded: [ // array with string elements
-        "X.com", 
-        "PayPal", 
-        "SpaceX", 
-        "The Musk Foundation", 
-        "Tesla", 
-        "SolarCity", 
-        "Neuralink", 
-        "The Boring Company"
-    ],
-    rolesInCompanies: { // objekts iekš objekta
-        twitter: {
-            role: "Potential business owner",
-            isFounder: false // boolean
-        },
-        tesla: {
-            role: "Business owner",
-            isFounder: true
-        }
-    }
-}
+  firstName: "Elon", // string
+  lastName: "Musk",
+  age: 50, // number
+  companiesFounded: [
+    // array with string elements
+    "X.com",
+    "PayPal",
+    "SpaceX",
+    "The Musk Foundation",
+    "Tesla",
+    "SolarCity",
+    "Neuralink",
+    "The Boring Company",
+  ],
+  rolesInCompanies: {
+    // objekts iekš objekta
+    twitter: {
+      role: "Potential business owner",
+      isFounder: false, // boolean
+    },
+    tesla: {
+      role: "Business owner",
+      isFounder: true,
+    },
+  },
+};
 
 // objekta īpašībām piekļūst norādot mainīgā nosaukumu (objekta nosaukumu) person.
 // Lai piekļūtu kādai no objekta īpašībām (mainīgie iekš objekta), rakstam pilnu hierarhijas ceļu līdz šai īpašībai.
 // piem., ar person.firstName var piekļūt objekta "person" īpašībai "firstName"
-console.log("Person full name:", person.firstName + person.lastName); 
+console.log("Person full name:", person.firstName + person.lastName);
```

Comment:
Šis, protams, strādā un ir ok.
Bet labi būtu izmantot "string literals" sintaksi:

`'${cv.firstName} ${cv.lastName}'`